Source/DeckViewExpander.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  DeckViewExpander.swift
+//  DeckTransition
+//
+//  Created by Rodrigo Kreutz on 18/09/2018.
+//
+
+import Foundation
+
+public protocol DeckViewExpander {
```

Comment:
If this protocol is going to be used on UIViewControllers only (no structs) it should be `: AnyObject`, right? :)

Reply:
Yep, makes sense

Reply:
Done aaa2059